apps/src/sharedComponents/footer/CopyrightDialog.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+import AccessibleDialog from '@cdo/apps/sharedComponents/AccessibleDialog';
+import SafeMarkdown from '@cdo/apps/templates/SafeMarkdown';
+import i18n from '@cdo/locale';
+
+import styles from '@cdo/apps/sharedComponents/footer/copyright-dialog.module.scss';
+
+export interface CopyrightDialogProps {
+  onClose: () => void;
+}
+
+const CopyrightDialog: React.FC<CopyrightDialogProps> = ({isOpen, onClose}) => {
+  const handleClose = () => {
+    if (typeof onClose === 'function') {
+      onClose();
+    }
+  };
+
+  const getDialogBody = () => {
+    return (
+      <div id="copyright-scroll-area">
+        <h3>{i18n.copyright()}</h3>
+        <hr />
+        <SafeMarkdown
+          markdown={i18n.copyright_thanks({
+            donors_link: 'https://code.org/about/donors',
+            partners_link: 'https://code.org/about/partners',
+            team_link: 'https://code.org/about/team',
+          })}
+        />
+        <p>{i18n.copyright_specialRecognition()}</p>
+        <SafeMarkdown
+          markdown={i18n.copyright_artFrom({
+            current_year: new Date().getFullYear(),
+          })}
+        />
+        <SafeMarkdown
+          markdown={i18n.copyright_codeLicense({
+            gnu_license_link:
+              'https://www.gnu.org/licenses/old-licenses/lgpl-2.1-standalone.html',
+          })}
+        />
+        <p>{i18n.copyright_builtOnGithub()}</p>
+        <a href="https://aws.amazon.com/what-is-cloud-computing">
+          <img
+            src="/shared/images/Powered-By_logo-horiz_RGB.png"
```

Comment:
Not sure if this is the right way of linking media, especially since the was picture does not render in Storybook, I just copied the reference from SmallFooter.jsx.

Reply:
From what i see in storybook, images sourced from `/shared/images/` do not load the images either.
Reference: CensusTeacherBanner.story.jsx
https://github.com/code-dot-org/code-dot-org/blob/ee1edeab6e4a5d47d618096ec2cddf4e3670b023/apps/src/templates/census2017/CensusTeacherBanner.jsx#L482
<img width="1070" alt="Screenshot 2024-08-26 at 10 52 56 AM" src="https://github.com/user-attachments/assets/f1632538-1be8-4228-849a-c46da98ad7f3">